node/src/components/network/event.rs

```diff
@@ -90,17 +96,33 @@ pub enum Event<P> {
     #[from]
     NetworkRequest {
         #[serde(skip_serializing)]
-        request: NetworkRequest<NodeId, P>,
+        request: Box<NetworkRequest<NodeId, P>>,
```

Comment:
Did we not manage to bring down the `NetworkRequest` down enough in size to avoid having to nest these in boxes?